docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/chip/chip.js

```diff
@@ -3,14 +3,15 @@
  * @file
  * Chip component.
  */
+
 function CivicChip(el) {
   if (el.getAttribute('data-chip') === 'true') {
     return;
   }
 
   this.el = el;
   this.el.setAttribute('data-chip', 'true');
-  this.dismissChip = this.el.querySelector('[data-chip-dismiss]');
+  this.dismissChip = this.el.querySelector('.civictheme-chip__icon--dismiss');
```

Comment:
please revert this. please add `data-chip-dismiss` attribute in twig if `is_dismissible` is true

Reply:
Updated